samtranslator/feature_toggle/feature_toggle.py

```diff
@@ -105,8 +107,11 @@ class FeatureToggleAppConfigConfigProvider(FeatureToggleConfigProvider):
 
     def __init__(self, application_id, environment_id, configuration_profile_id):
         FeatureToggleConfigProvider.__init__(self)
-        self.app_config_client = boto3.client("appconfig")
         try:
+            # Lambda function has 120 seconds limit
+            # (5 + 25) * 2, 60 seconds maximum timeout duration
+            client_config = Config(connect_timeout=5, read_timeout=25, retries={"total_max_attempts": 2})
+            self.app_config_client = boto3.client("appconfig", config=client_config)
```

Comment:
Why was this moved under the `try`? If this fails the error will be silently ignored.

Reply:
We shouldn't fail a request just because an error is caused by feature toggle. Proceed with empty config on errors should be the default.